index.js

```diff
@@ -4,8 +4,14 @@ import PropTypes from 'prop-types';
 import terminalLink from 'terminal-link';
 
 const Link = props => {
+	const options = {
+		fallback: (text, url) => {
+			return props.fallback ? `${text} (\u200B${url}\u200B)` : text;
```

Comment:
Happy to open a PR to `terminal-link` to export the default fallback function, if we want to avoid duplicating it here.

Reply:
I would make `terminal-link` support {fallback: false} to disable the fallback. Then we don’t need to export it. What do you think?

Reply:
Down for that, too! Can open a PR in a sec.

Reply:
@colinking Friendly bump :)

Reply:
Thanks for the bump @sindresorhus! Opened a PR here: https://github.com/sindresorhus/terminal-link/pull/13